test/include/util/transaction_test_util.h

```diff
@@ -107,6 +107,11 @@ class RandomWorkloadTransaction {
 class LargeTransactionTestObject {
  public:
   /**
+   * Builder class for LargeTransactionTestObject
+   */
+  class Builder;
+
+    /**
```

Comment:
Usually the constructor of a class is made private to avoid unwanted invocation with a builder. You should also convert all test cases to use the builder after you are done with that.